src/app/dev/DevToys.Tools/Tools/Converters/JsonTable/JsonTableConverterGuiTool.cs

```diff
@@ -0,0 +1,210 @@
+﻿using System.Data;
+using DevToys.Tools.Tools.Converters.JsonTable;
+using Microsoft.Extensions.Logging;
+using static DevToys.Tools.Helpers.JsonTableHelper;
+
+namespace DevToys.Tools.Tools.Converters.JsonYaml;
+
+[Export(typeof(IGuiTool))]
+[Name("JsonTableConverter")]
+[ToolDisplayInformation(
+    IconFontName = "DevToys-Tools-Icons",
+    IconGlyph = '\u0109',
```

Comment:
Note to myself: will need a new icon. But it isn't blocking this PR. We can merge it and change the icon later.

Reply:
Created an issue to track this remaining work: https://github.com/veler/DevToys/issues/1005